src/components/vr_cooperation/src/on_register_service_notification.cc

```diff
@@ -30,50 +30,44 @@
  POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "vr_cooperation/commands/register_service_request.h"
+#include "vr_cooperation/commands/on_register_service_notification.h"
 
-#include "functional_module/function_ids.h"
 #include "utils/logger.h"
-#include "vr_cooperation/vr_module_constants.h"
 
 namespace vr_cooperation {
 
 namespace commands {
 
-CREATE_LOGGERPTR_GLOBAL(logger_, "RegisterServiceRequest")
+CREATE_LOGGERPTR_GLOBAL(logger_, "VRCooperation")
 
-RegisterServiceRequest::RegisterServiceRequest(
+OnRegisterServiceNotification::OnRegisterServiceNotification(
       VRModule* parent,
       const application_manager::MessagePtr& message)
-  : BaseCommandRequest(parent, message) {
+  : GPBNotification(parent, message) {
 }
 
-RegisterServiceRequest::~RegisterServiceRequest() {
+OnRegisterServiceNotification::~OnRegisterServiceNotification() {
 }
 
-void RegisterServiceRequest::Execute() {
+void OnRegisterServiceNotification::Execute() {
   LOG4CXX_AUTO_TRACE(logger_);
 
-  Json::Value params;
-  Json::Reader reader;
-  reader.parse(json_message_->json_message(), params);
+  vr_hmi_api::ServiceMessage service_message;
+  service_message.set_rpc(vr_hmi_api::ON_REGISTER);
 
-  std::string result_code = result_codes::kSuccess;
-  bool success = true;
-  std::string info;
+  vr_hmi_api::OnRegisterServiceNotification onregister_notification;
+  int32_t app_id = message()->connection_key();
+  app_id == parent()->default_app_id() ?
```

Comment:
In this case it is better to use general condition operator `if .. else ..` in my opinion.